staging/src/k8s.io/pod-security-admission/test/fixtures_sysctls.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/pod-security-admission/api"
+)
+
+/*
+TODO: include field paths in reflect-based unit test
+
+podFields: []string{
+	`spec.securityContext.sysctls.name`,
+},
+*/
+
+func init() {
+	fixtureData_1_0 := fixtureGenerator{
+		expectErrorSubstring: "forbidden sysctl",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureSecurityContext(p)
+			return []*corev1.Pod{
+				// security context with no sysctls
+				tweak(p, func(p *corev1.Pod) { p.Spec.SecurityContext.Sysctls = nil }),
+				// sysctls with name=""
+				tweak(p, func(p *corev1.Pod) { p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{} }),
+				// sysctls with name="kernel.shm_rmid_forced"
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{{Name: "kernel.shm_rmid_forced", Value: "0"}}
+				}),
+				// sysctls with name="net.ipv4.ip_local_port_range"
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{{Name: "net.ipv4.ip_local_port_range", Value: "1024 65535"}}
+				}),
+				// sysctls with name="net.ipv4.tcp_syncookies"
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{{Name: "net.ipv4.tcp_syncookies", Value: "0"}}
+				}),
+				// sysctls with name="net.ipv4.ping_group_range"
+				tweak(p, func(p *corev1.Pod) {
+					p.Spec.SecurityContext.Sysctls = []corev1.Sysctl{{Name: "net.ipv4.ping_group_range", Value: "1 0"}}
+				}),
+			}
+		},
+		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureSecurityContext(p)
```

Comment:
since we don't need to exercise the container securityContext fields for sysctls, consider only setting `podSpec.SecurityContext` here if nil to clean up the rendered fixtures (don't forget to regenerate them)

Reply:
Done. Hope i understand it right here.